packages/json-api/src/request.ts

```diff
@@ -41,7 +41,7 @@ const options = findRecord('ember-developer', '1', { include: ['pets', 'friends'
     op: 'findRecord';
     records: [{ type: 'ember-developer', id: '1' }]
   }
-*\/
+*\
```

Comment:
I wonder if we need `\/*` and `\*/` instead

Reply:
I think this should work.

That's how we do it in other files for example [here](https://github.com/emberjs/data/blob/3ec0359017d321d43d914f4fffda51c0672f58b5/packages/rest/src/request.ts#L34)

Reply:
If we want to change it to `\*/`.  we can do it.